eth/subs/subs_set.go

```diff
@@ -59,7 +59,9 @@ func (s *EthSubscriptionSet) EmitEvent(data types.EventData) error {
 	if err != nil {
 		return errors.Wrapf(err, "marshaling event %v", data)
 	}
-	s.logsHub.Publish(pubsub.NewMessage(string(ethMsg), eth.EncEvent(data)))
+	jLog := eth.EncEvent(data)
+	jLog.BlockTime = eth.EncInt(data.BlockTime)
```

Comment:
Any particular reason you didn't set the block time in `eth.EncEvent` itself?

Reply:
When writing out the list of events in a transaction receipt, the blocktime is left out.

Reply:
Right, but why?

Reply:
Not sure. My guess FWIW is that there are technical difficulties in go-ethereum in returning the blocktime with receipts, possibly because ethereum tends to rerun transactions to get receipts rather than save them.

https://github.com/loomnetwork/go-ethereum/blob/loomchain/core/types/receipt.go#L46
https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_gettransactionreceipt

Reply:
The block time on events is a Loom extension, so what go-ethereum does in this case doesn't matter. The old `/query` endpoint for fetching receipts returns the block time in the event logs, so we should do the same with the `/eth` endpoint.

Reply:
Ah! OK! I'll add the block time to logs internal to receipts as well.